### DISCUSSION TITLE
Helpful tip for discovering devices if they're not being detected

### DISCUSSION THREAD

**Discussion by jmporter34** [krahabb/meross_lan#161] [General]
Hi, thanks @krahabb for sharing your work.  Per your instructions, I downloaded your integration with HACS -> Integrations, installed it and restarted HA.  But I wasn't seeing any of my Meross devices pop up on Configuration -> Integrations as you suggested they would.  After re-downloading your integration, re-installing it, restarting HA a few times with no luck, I remembered having a similar issue when I started working with the Aqara integration.  It wasn't detecting any of my Aqara hubs as it was supposed to.  The culprit there turned out to be that I had added them in the Apple Home app on my phone and had never removed them.  After removing the Aqara hubs from Apple Home, HA detected them.  That turned out to do the trick here as well.  I removed my Meross plugs and light switches from Apple Home, and they immediately popped up on Configuration -> Integrations.  Looking forward to using your integration, and again thanks for sharing your work !

**Comment by krahabb** [comment]
Thank you for your contribution. I'm curious to understand why this happens (need for removal from Apple Home) since Meross HomeKit devices should also be working in 'legacy' mode (i.e. Meross App-Cloud and eventually meross_lan) but maybe things are really running 'fast forward' and I don't have un updated picture about current Meross implementations.
Nice point anyway..I'll try to keep it alive for similar issues or so
